CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```

Comment:
I feel like this is something which ought to have been implicit from the previous PR as it is more of a fix, but since it's not a fix of something already released I've decided not to mention it.